packages/core/src/routes/connector.ts

```diff
@@ -48,9 +48,9 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         params: { id },
         body: { enabled },
       } = ctx.guard;
-      await findConnectorById(id);
-      await updateConnector({ set: { enabled }, where: { id } });
-      ctx.body = { enabled };
+      const { metadata } = await getConnectorInstanceById(id);
+      const connector = await updateConnector({ set: { enabled }, where: { id } });
+      ctx.body = { ...connector, metadata };
```

Comment:
Would you let me know why this API doesn't call `validateConfig()` while the other one does? Thanks.
I'm a bit confused.

Reply:
as you can see from the method name `validateConfig`, it is used to check the validity of property `config`.
`config` is a property that varies by different connectors. as a result, `validateConfig` should be customized for each connector.
`enabled` is a boolean-type property that is easy to be checked.
![image](https://user-images.githubusercontent.com/15182327/150903074-9dc293e4-d339-43bd-a712-9db4765988cd.png)
the reason why we separate props `enabled` and `config` into two routes is to enable the situation that we may update each prop independently.

Reply:
OK, got it. Thanks for the explanation.